logging/google/cloud/logging/client.py

```diff
@@ -327,3 +323,19 @@ def setup_logging(self, log_level=logging.INFO,
         handler = self.get_default_handler()
         setup_logging(handler, log_level=log_level,
                       excluded_loggers=excluded_loggers)
+
+    def enable_shutdown_logging(self, thread_dump=True):
+        """Enable shutdown report logging.
+
+        This method installs a SIGTERM handler that will report various
+        application metrics to Stackdriver Logging.
+
+        Currently the only supported option is stacktrace logging.
+
+        :type log_level: bool
+        :param log_level: (Optional) When true, on SIGTERM the application
+                          will log the stacktrace of all running threads.
+        """
+        if thread_dump:
```

Comment:
I am confused by the `thread_dump` argument. If it is `False`, the method no-ops? Why have the argument at all?